src/templates/next/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,298 @@
+        {siteNavItems.map((item) => (
+          <Link title={item.title} url={item.url} />
+        ))}
+      </div>
+      <div className="flex flex-col gap-5">
+        {customNavItems?.map((item) => (
+          <Link title={item.title} url={item.url} />
+        ))}
+      </div>
+    </div>
+  )
+}
+
+const SocialMediaSection = ({
+  socialMediaLinks,
+}: Pick<FooterProps, "socialMediaLinks">) => {
+  return (
+    <div className="flex flex-col gap-5">
+      <h5 className={Heading[5]}>Reach us</h5>
+      <div className="flex flex-row gap-7 flex-wrap">
+        {socialMediaLinks?.map((link) => {
+          const Icon = SocialMediaTypeToIconMap[link.type]
+          return (
+            <a
+              key={link.url}
+              href={link.url}
+              target="_blank"
+              rel="noopener noreferrer nofollow"
+            >
+              <Icon size={26} />
+            </a>
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+const ContactUsSection = ({
+  contactUsLink,
+  feedbackFormLink,
+}: Pick<FooterProps, "contactUsLink" | "feedbackFormLink">) => {
+  return (
+    <div className={`flex flex-col gap-3 ${Paragraph[2]}`}>
+      {contactUsLink && <Link title="Contact Us" url={contactUsLink} />}
+      {feedbackFormLink && (
+        <Link title="Feedback Form" url={feedbackFormLink} />
+      )}
+    </div>
+  )
+}
+
+const ReachUsSection = ({
+  socialMediaLinks,
+  contactUsLink,
+  feedbackFormLink,
+}: Pick<
+  FooterProps,
+  "socialMediaLinks" | "contactUsLink" | "feedbackFormLink"
+>) => {
+  return (
+    <div className="flex flex-col gap-6 lg:gap-14 lg:w-fit">
+      <SocialMediaSection socialMediaLinks={socialMediaLinks} />
+      <ContactUsSection
+        contactUsLink={contactUsLink}
+        feedbackFormLink={feedbackFormLink}
+      />
+    </div>
+  )
+}
+
+const LegalSection = ({
+  agencyName,
+  isGovernment,
+  lastUpdated,
+  privacyStatementLink,
+  termsOfUseLink,
+}: Pick<
+  FooterProps,
+  | "agencyName"
+  | "isGovernment"
+  | "lastUpdated"
+  | "privacyStatementLink"
+  | "termsOfUseLink"
+>) => {
+  return (
+    <div className="flex flex-col gap-8 lg:gap-2">
+      <p className={`text-content-inverse-light ${Paragraph[2]}`}>
+        © {new Date().getFullYear()}{" "}
+        {isGovernment ? "Government of Singapore" : agencyName}, last updated{" "}
+        {lastUpdated}
+      </p>
+      <div
+        className={`flex flex-col gap-3 lg:flex-row lg:gap-8 ${Paragraph[2]}`}
+      >
+        {isGovernment && (
+          <Link
+            title="Report Vulnerability"
+            url="https://tech.gov.sg/report_vulnerability"
+          />
+        )}
+        {privacyStatementLink && (
+          <Link title="Privacy Statement" url={privacyStatementLink} />
+        )}
+        {termsOfUseLink && <Link title="Terms of Use" url={termsOfUseLink} />}
+        {isGovernment && (
+          <Link title="Reach" url={"https://www.reach.gov.sg"} />
+        )}
```

Comment:
double checking if this makes sense - hiding Report vul and Reach for non gov sites
how do we do it currently?